### PR TITLE
globalProperties are not passed in the very first page

### PR DESCRIPTION
**Pull request by liulianci**

### CODE REVIEW

File: ios/ElectrodeApiImpl/APIImpls/Core/ENCoreDelegate.swift [hunk 1]
```diff
@@ -26,7 +26,10 @@ import UIKit
         if let miniAppVC = viewController as? MiniAppNavViewController {
             self.viewIdentifier = UUID().uuidString
             let viewIdProperty = [ENCoreDelegate.KEY_UNIQUE_VIEW_IDENTIFIER: viewIdentifier] as [AnyHashable : Any]
-            self.createView(name: miniAppVC.miniAppName, properties: combineRouteData(dictionary1: miniAppVC.properties, dictionary2: viewIdProperty))
+            let extraProperties = miniAppVC.pushToExistingViewController ?
+                combineRouteData(dictionary1: viewIdProperty, dictionary2: miniAppVC.globalProperties) :
```
Comment (deepueg) [inline ios/ElectrodeApiImpl/APIImpls/Core/ENCoreDelegate.swift:30]: Why do we need to add this to the core delegate? globalProperties is only applicable to those who use navigate, right ?

File: ios/ElectrodeApiImpl/APIImpls/Navigation/MiniAppNavViewController.swift [hunk 2]
```diff
@@ -23,6 +23,7 @@ open class MiniAppNavViewController: UIViewController, ENNavigationProtocol {
     public var finish: Payload?
     public var delegate: ENNavigationDelegate?
     public var navigateWithRoute: NavigateWithRoute?
+    public var pushToExistingViewController: Bool = true
```
Comment (deepueg) [inline ios/ElectrodeApiImpl/APIImpls/Navigation/MiniAppNavViewController.swift:26]: adding this to a`MiniAppNavViewController.swift` limits the ability for the apps to create their own VCs with the delegates. See that there are also other methods added to the MiniAppVC. Let's have a discussion.

Comment (liulianci) [reply ios/ElectrodeApiImpl/APIImpls/Navigation/MiniAppNavViewController.swift:26]: This boolean value is required when pushing a miniappNavViewController on top of the existing navigationController. It is still okay if someone else wants to create their own VC and use ENCoreDelegate, ENCoreDelegate will check if the VC is a instance of MiniAppNavViewController, if VC is not a instance of MiniAppNavViewController, this boolean value is not being used.

File: ios/ElectrodeApiImpl/APIImpls/Core/ENCoreDelegate.swift (hunk 1 above)
Comment (liulianci) [reply ios/ElectrodeApiImpl/APIImpls/Core/ENCoreDelegate.swift:30]: we'll need this if we push a MiniappNavViewController on top of a existing NavigationController

**Pull request merged**